src/connection.ts

```diff
@@ -365,7 +374,7 @@ export class Connection extends EventEmitter {
     await this.sendConnectionClose()
     this.safeEmit('end')
     this.safeEmit('close')
-    clearTimeout(this.idleTimer)
+    this.stopTimers()
```

Comment:
Can `stopTimers` be removed here and at the end of `destroy` since both invoke `closeHandler` which calls `stopTimers`?

Reply:
I think generally the `closeHandler` will have been removed by the time `end`/`destroy` are called. `closeHandler` is removed as soon as the `'connect'` event is emitted during connection setup.